packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -49,6 +50,18 @@ const ConnectorDetails = () => {
   const api = useApi();
   const navigate = useNavigate();
   const theme = useTheme();
+  const isSocial = data?.type === ConnectorType.Social;
+  const [isDeleteAlertOpen, setIsDeleteAlertOpen] = useState(false);
+
+  const onDeleteClick = async () => {
+    if (!isSocial || !inUse) {
```

Comment:
non-social connectors should also have alert?

Reply:
Per offline discussion, we only show this alert on the social connectors page.